### PR TITLE
What's new in v4

### PR DESCRIPTION
**Pull request by NolwennP**
Migrate the whole v4 release note in a web format in the help center under the `v4/updates` url.

### CODE REVIEW

File: content/updates/data-quality-insights/1-data-quality-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# A data quality dashboard 
+
+With the Data Quality Insights feature, you can easily measure the quality of your product data and see at a glance how it measures up in a dedicated dashboard.
```
Comment (LaureBro) [inline content/updates/data-quality-insights/1-data-quality-dashboard.md:3]:
```suggestion
With the Data Quality Insights feature, you can easily **measure the quality of your product data and monitor it at a glance** in a dedicated dashboard.
```

File: content/updates/data-quality-insights/1-data-quality-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+This dashboard uses a grading system based on enrichment and consistency. The enrichment axis checks required and optional attributes, while the consistency axis monitors spelling and case formatting.
```
Comment (LaureBro) [inline content/updates/data-quality-insights/1-data-quality-dashboard.md:5]:
```suggestion
This dashboard uses a **grading system based on enrichment and consistency**. The enrichment axis checks required and optional attributes, while the consistency axis monitors spelling and case formatting.
```

File: content/updates/data-quality-insights/1-data-quality-dashboard.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+In addition to a global product catalog view, you can focus on the quality of a specific category or family to better prioritize your tasks.
```
Comment (LaureBro) [inline content/updates/data-quality-insights/1-data-quality-dashboard.md:7]:
```suggestion
In addition to a global product catalog view, you can **focus on the quality of a specific category or family to better prioritize your tasks**.
```

File: content/updates/index.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+  "asset-manager": {
+    "title": "The Asset Manager",
+    "description": "Digital assets are a critical element in Product Experience Management. By providing additional information about the product, they help brands tell a story to the customer, increasing emotional impact. In Akeneo PIM 4.0, we have enhanced the ability to manage digital assets with a new feature, known as the Asset Manager, designed to support the unique needs of B2C and B2B enterprises alike. The new Asset Manager delivers a significantly better user experience and offers greater flexibility and more connectivity.",
```
Comment (LaureBro) [inline content/updates/index.json:4]:
```suggestion
    "description": "Digital assets are a **critical element in Product Experience Management**. By providing additional information about the product, they **help brands tell a story to the customer, increasing emotional impact**. In Akeneo PIM 4.0, we have enhanced the ability to manage digital assets with a new feature, known as the Asset Manager, designed to support the unique needs of B2C and B2B enterprises alike. The new Asset Manager **delivers a significantly better user experience and offers greater flexibility and more connectivity**.",
```

Comment (LaureBro) [reply content/updates/index.json:4]: Est-ce qu'on peut mettre du bold ici ?

File: content/updates/data-quality-insights/1-data-quality-dashboard.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# A data quality dashboard
```
Comment (LaureBro) [inline content/updates/data-quality-insights/1-data-quality-dashboard.md:1]:
```suggestion
# Monitor your data quality in a dashboard 
```

File: content/updates/asset-manager/2-3rd-party-media.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# Easily retrieve 3rd party media
```
Comment (LaureBro) [inline content/updates/asset-manager/2-3rd-party-media.md:1]:
```suggestion
# New API endpoints to easily retrieve 3rd party media
```

File: content/updates/asset-manager/1-greater-flexibility.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# With greater flexibility come greater possibilities
```
Comment (LaureBro) [inline content/updates/asset-manager/1-greater-flexibility.md:1]:
```suggestion
# More flexibility to manage your assets
```

File: content/updates/asset-manager/5-asset-data-contextualization.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+# Asset data contextualization
```
Comment (LaureBro) [inline content/updates/asset-manager/5-asset-data-contextualization.md:1]:
```suggestion
# Contextualize your asset information
```

File: content/updates/asset-manager/6-transformation-rules.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
+# Asset transformation rules
```
Comment (LaureBro) [inline content/updates/asset-manager/6-transformation-rules.md:1]:
```suggestion
# Improved asset transformation rules
```

File: content/updates/asset-manager/7-other-features.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
+# Other notable features
```
Comment (LaureBro) [inline content/updates/asset-manager/7-other-features.md:1]:
```suggestion
# Additional Asset Manager features
```

File: content/updates/data-quality-insights/2-quick-action.md [hunk 11]
```diff
@@ -0,0 +1,8 @@
+# Fix poor data quality with quick action
```
Comment (LaureBro) [inline content/updates/data-quality-insights/2-quick-action.md:1]:
```suggestion
# How does it work?
```

File: content/updates/connections/2-connections-in-one-place.md [hunk 12]
```diff
@@ -0,0 +1,9 @@
+# All your connections in one place
```
Comment (LaureBro) [inline content/updates/connections/2-connections-in-one-place.md:1]:
```suggestion
# All your connections in one single place
```

File: content/updates/franklin-insights/1-suggested-mapping.md [hunk 13]
```diff
@@ -0,0 +1,8 @@
+# Suggested mapping
```
Comment (LaureBro) [inline content/updates/franklin-insights/1-suggested-mapping.md:1]:
```suggestion
# New suggested mapping feature
```

File: content/updates/franklin-insights/2-franklin-key-figures.md [hunk 14]
```diff
@@ -0,0 +1,3 @@
+# Franklin key figures
```
Comment (LaureBro) [inline content/updates/franklin-insights/2-franklin-key-figures.md:1]:
```suggestion
# Improvements on Franklin key figures dashboard
```

File: content/updates/asset-manager/6-transformation-rules.md [hunk 15]
```diff
@@ -0,0 +1,6 @@
...
+
+The new Asset Manager also offers additional flexibility in terms of asset transformations. You can, of course, keep your asset images as they are, distributing them in their original format without any transformation — but
+many times you need to transform the digital asset for different channel requirements.
```
Comment (NolwennP) [inline content/updates/asset-manager/6-transformation-rules.md:4]:
```suggestion
The new Asset Manager also offers additional flexibility in terms of asset transformations. You can, of course, keep your asset images as they are, distributing them in their original format without any transformation — but many times you need to transform the digital asset for different channel requirements. 
```

[the remark this answers is not in the document]
Comment (NolwennP) [reply content/updates/asset-manager/6-transformation-rules.md:4]: Yes, you're right!

File: content/updates/index.json (hunk 4 above)
Comment (NolwennP) [reply content/updates/index.json:4]: Unfortunately, no. :(

File: content/updates/asset-manager/6-transformation-rules.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (NolwennP) [reply content/updates/asset-manager/6-transformation-rules.md]: Fixed!

**Pull request merged**